data/js/tomahawk.js

```diff
@@ -49,9 +51,9 @@ RSVP.on('error', function (reason) {
         resolverName = Tomahawk.resolver.instance.settings.name + " - ";
     }
     if (reason) {
-        console.error(resolverName + 'Uncaught error:', reason);
+        Tomahawk.error(resolverName + 'Uncaught error:', reason);
     } else {
-        console.error(resolverName + 'Uncaught error: error thrown from RSVP but it was empty');
+        Tomahawk.error(resolverName + 'Uncaught error: error thrown from RSVP but it was empty');
```

Comment:
On desktop it simply hands the object over to console.error. So it shouldn't change anything there